### DISCUSSION TITLE
coder.ceval simulink example and support

### DISCUSSION THREAD

**Discussion by jtcook2** [casadi/casadi#3256] [Q&A]
There is an existing s function example in a blog post using the generated c code. It looks like it should also be possible to use the coder.ceval interface in a matlab function block in simulink.

Is there an existing example of this or has anyone done this successfully?

**Comment by jgillis** [comment]
It looks like that should be feasible indeed; have not seen it in the wild.
If you figure it out, that would be nice material for a new blog post..
